src/wagtailtrans/migrations/0001_initial.py

```diff
@@ -32,9 +32,9 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='TranslatablePage',
             fields=[
-                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+                ('translatable_page_ptr', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, related_name='+', serialize=False, to='wagtailcore.Page')),
```

Comment:
Yes I know, didn't have the time to look into this yet. I think this part needs to be solved in the project itself, don't think django's migrations can fix this easily. Still want to make a migration guide where this should be documented..

Any idea's on how to solve the issue?